core/src/main/java/com/linecorp/armeria/client/Clients.java

```diff
@@ -427,6 +430,7 @@ public static SafeCloseable withHttpHeader(CharSequence name, String value) {
      *
      * @see #withHttpHeaders(Consumer)
      */
+    @MustBeClosed
```

Comment:
Is it possible to annotate `SafeCloseable` with `@MustBeClosed` so that we do not have to for all `SafeCloseable`-returning methods?

Reply:
The target of `@MustBeclosed` is `CONSTRUCTOR` and `METHOD` so I guess it didn't work. 🤔

Reply:
https://github.com/google/error-prone/issues/1708 Filed an issue.

Reply:
Thanks, @minwoox